lib/web_ui/lib/semantics.dart

```diff
@@ -189,6 +190,7 @@ class SemanticsFlag {
   static const SemanticsFlag hasToggledState = SemanticsFlag._(_kHasToggledStateIndex);
   static const SemanticsFlag isToggled = SemanticsFlag._(_kIsToggledIndex);
   static const SemanticsFlag hasImplicitScrolling = SemanticsFlag._(_kHasImplicitScrollingIndex);
+  static const SemanticsFlag isMixedCheck = SemanticsFlag._(_kIsMixedCheckIndex);
```

Comment:
We may need a better name. maybe `isCheckStateMixed`

Reply:
Maybe `isMixedCheckState`? But it's a bit at odds with existing naming convention typically `hasFooState` and `isFoo`.